src/Symfony/Component/HttpKernel/Profiler/Profiler.php

```diff
@@ -60,6 +60,16 @@ public function enable()
         $this->enabled = true;
     }
 
+    /**
+     * Gets a value indicating whether the profiler is enabled.
+     *
+     * @return bool True if the profiler is enabled, otherwise false
+     */
```

Comment:
I would remove the phpdoc block as if doesn't add anything that cannot be understood with the method name and signature.

Reply:
This breaks the convention used throughout the rest of the file, but sure.